src/fluidplayer.js

```diff
@@ -2729,48 +2729,52 @@ const fluidPlayerClass = function () {
     };
 
     self.setPersistentSettings = () => {
-        if (!(typeof (Storage) !== 'undefined' && typeof (localStorage) !== 'undefined')) {
-            return;
-        }
-
-        // See https://github.com/fluid-player/fluid-player/issues/271
-        const testKey = '_fp_storage_enabled', storage = localStorage;
         try {
-            storage.setItem(testKey, '1');
-            storage.removeItem(testKey);
-        } catch (error) {
-            return false;
-        }
+            if (!(typeof (Storage) !== 'undefined' && typeof (localStorage) !== 'undefined')) {
+                return;
+            }
```

Comment:
I think it's overkill moving everything in the function to a try..catch, if this is the line that is breaking the code. I would move this to the try..catch, it would make sense as this code is intended to verify if it's possible to use localStorage. Alternatively you can add the try..catch to the function call, and add a debug message with the error as to not "do nothing" if it catches an error.